mlflow/gateway/providers/mlflow.py

```diff
@@ -0,0 +1,143 @@
+from fastapi import HTTPException
+from fastapi.encoders import jsonable_encoder
+import json
+
+from .base import BaseProvider
+from .utils import send_request
+from ..config import RouteConfig, MlflowModelServingConfig
+from ..schemas import completions, chat, embeddings
+
+
+class MlflowModelServingProvider(BaseProvider):
+    def __init__(self, config: RouteConfig) -> None:
+        super().__init__(config)
+        if config.model.config is None or not isinstance(
+            config.model.config, MlflowModelServingConfig
+        ):
+            raise TypeError(f"Invalid config type {config.model.config}")
+        self.mlflow_config: MlflowModelServingConfig = config.model.config
+        self.headers = {"Content-Type": "application/json"}
+
+    @staticmethod
+    def process_payload(payload, key):
+        payload = jsonable_encoder(payload, exclude_none=True)
+
+        input_data = payload.pop(key)
```

Comment:
```suggestion
        input_data = payload.pop(key, None)
```

`pop(key)` looks unsafe. Can we add `None`?